### PR TITLE
utilize new, minor `pacta.data.preparation` functions

### PR DESCRIPTION
**Pull request by cjyetman**
- [x] depends on https://github.com/RMI-PACTA/pacta.data.preparation/pull/348

### CODE REVIEW

File: run_pacta_data_preparation.R [hunk 1]
```diff
@@ -314,12 +314,10 @@ invisible(gc())
...
 factset_entity_id__ar_company_id <-
-  readr::read_csv(ar_company_id__factset_entity_id_path, col_types = "c") %>%
-  select(
-    factset_entity_id = "factset_id",
-    ar_company_id = "company_id"
-  ) %>%
-  distinct()
+  pacta.data.preparation::prepare_factset_entity_id__ar_company_id(
+    readr::read_csv(ar_company_id__factset_entity_id_path, col_types = "c")
+  )
```
Comment (jdhoffa) [inline run_pacta_data_preparation.R:319]: NIT/ style: maybe pipe this for clarity? I always find it a little difficult to read when the first argument of a function is another function with multiple arguments

File: run_pacta_data_preparation.R [hunk 2]
```diff
@@ -422,17 +411,15 @@ financial_data <- readRDS(file.path(config[["data_prep_outputs_path"]], "financi
 entity_info <- readRDS(file.path(config[["data_prep_outputs_path"]], "entity_info.rds"))
 
 factset_entity_id__ar_company_id <-
-  entity_info %>%
-  select(factset_entity_id, ar_company_id) %>%
-  filter(!is.na(ar_company_id))
+  pacta.data.preparation::prepare_factset_entity_id__ar_company_id(
+    readr::read_csv(ar_company_id__factset_entity_id_path, col_types = "c")
+  )
```
Comment (jdhoffa) [inline run_pacta_data_preparation.R:416]: NIT/styel: same as above

File: run_pacta_data_preparation.R [hunk 3]
```diff
@@ -441,12 +428,10 @@ invisible(gc())
...
-  readRDS(file.path(config[["data_prep_outputs_path"]], "masterdata_ownership_datastore.rds")) %>%
-  filter(year %in% relevant_years) %>%
-  select(ar_company_id = id, ald_sector) %>%
-  distinct() %>%
-  group_by(ar_company_id) %>%
-  summarise(sectors_with_assets = paste(unique(ald_sector), collapse = " + "))
+  pacta.data.preparation::prepare_ar_company_id__sectors_with_assets__ownership(
+    readRDS(file.path(config[["data_prep_outputs_path"]], "masterdata_ownership_datastore.rds")),
+    relevant_years
+  )
```
Comment (jdhoffa) [inline run_pacta_data_preparation.R:434]: same NIT as above

File: run_pacta_data_preparation.R [hunk 4]
```diff
@@ -467,12 +452,10 @@ invisible(gc())
...
-  readRDS(file.path(config[["data_prep_outputs_path"]], "masterdata_debt_datastore.rds")) %>%
-  filter(year %in% relevant_years) %>%
-  select(ar_company_id = id, ald_sector) %>%
-  distinct() %>%
-  group_by(ar_company_id) %>%
-  summarise(sectors_with_assets = paste(unique(ald_sector), collapse = " + "))
+  pacta.data.preparation::prepare_ar_company_id__sectors_with_assets__debt(
+    readRDS(file.path(config[["data_prep_outputs_path"]], "masterdata_debt_datastore.rds")),
+    relevant_years
+  )
```
Comment (jdhoffa) [inline run_pacta_data_preparation.R:458]: same NIT

**Review by jdhoffa — APPROVED** [review]
some NB/ NIT/ style comments, but LGTM

File: run_pacta_data_preparation.R (hunk 1 above)
Comment (cjyetman) [reply run_pacta_data_preparation.R]: done in all cases

**Review by jdhoffa — APPROVED** [review]

**Pull request merged**